creditcalc.py

```diff
@@ -9,36 +9,32 @@
 parser.add_argument("--payment", type=float)
 args = parser.parse_args()
 
-if args.interest is None:
-    print("Incorrect parameters.")
-    exit()
+if (args.type != "diff" and args.type != "annuity" or args.interest is None):
```

Comment:
Overall ensuring that each program flow has the required arguments would take more than this. Also it is not clear what one needs to provide for the different calc modes. Anyways, I presume this is out of the scope of this task. I have unified here some of the checks.